codes/kotlin/chapter_array_and_linkedlist/list.kt

```diff
@@ -0,0 +1,65 @@
+/**
+ * File: list.kt
+ * Created Time: 2024-01-25
+ * Author: curtishd(1023632660@qq.com)
+ */
+
+package chapter_array_and_linkedlist
+
+/* Driver Code */
+fun main() {
+    // 注意数组的元素类型是 int[] 的包装类 Integer[]
```

Comment:
`Integer[]` is the language-specific(Java) comment

Reply:
ok